Source/Core/Core/Boot/Boot_BS2Emu.cpp

```diff
@@ -163,6 +163,11 @@ bool CBoot::RunApploader(bool is_wii, const DiscIO::VolumeDisc& volume)
   // return
   PC = PowerPC::ppcState.gpr[3];
 
+  if (volume.GetVolumeType() == DiscIO::Platform::Triforce)
+  {
+    Memory::Memset(0, 0, 12);
```

Comment:
Maybe add an explanatory comment about why this is necessary. This looks pretty bizarre in isolation